int32 level
string message
string date
string node_name
string file
string code_line
